### DISCUSSION TITLE
How do I uninstall the Open Source Mirth engine?

### DISCUSSION THREAD

**Discussion by JonEBee** [nextgenhealthcare/connect#5034] [Q&A]
I have an installation on my personal laptop, inside a VBox CentOS server.  Tried my own custom install and haven't done too well by it, so I want to remove the installation and start over.

Uninstalling or removing the engine software doesn't appear to exist anywhere?  Am I missing something?  

Thanks in advance for your thoughts.

**Comment by pacmano1** [comment]
If you tried you own custom install and don't share how you did the install it is sort of hard to answer your question.

If you extracted the tar.gz file, you can just delete the entire directory it created.

If you used the installer for linux, there is an "uninstall" program in the same place as mcserver (e.g. /usr/local/mirthconnect).

<img width="838" alt="image" src="https://user-images.githubusercontent.com/44065187/154773140-dd343670-4373-47a8-8ae4-9ae298c7a0bf.png">

**Reply by pacmano1** [reply]
You will also need to remove the directories you specified when the installer asked you for them.  i.e. the logs and appdata dirs.

**Comment by cturczynskyj** [comment]
If you used RPM to install you can use `rpm -e mirthconnect` to uninstall.

If you used YUM to install (something like `yum localinstall mirthconnect-3.12.0.b2650-linux.rpm`) then you would use `yum remove mirthconnect`.
